src/Components/LetterBuilder/Letter/Letter.tsx

```diff
@@ -59,15 +67,14 @@ const LetterHeader: React.FC<LetterData> = ({ letterData }) => {
         )}
         {toTitleCase(ud.city)}, {ud.state} {ud.zip_code}
         <br />
-        {ud.email}
+        {formatPhoneNumber(ud.phone_number)}
```

Comment:
shouldn't this still be the user email, since in the disclaimer we say we won't share your number, but do say we'll include your email. Also I think we discussed with experts not including phone since we don't want to encourage/facilitate communication with the landlord over the phone - only in writing. 
 ```suggestion
        {ud.email}
```